### PR TITLE
Add readiness and liveness probs to other pods

### LINKED ISSUE #1723: Add readiness and liveness probs to other prods
In https://github.com/fybrik/fybrik/pull/1703 we added readiness and liveness probs to the manager pod. We need to add similar probes to other pods.

### PR DESCRIPTION
**Pull request by revit13**
/closes #1723

### CODE REVIEW

File: charts/fybrik/values.yaml [hunk 1]
```diff
@@ -341,6 +364,11 @@ opaServer:
   # ref: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-1
   containerSecurityContext: {}
 
+  enableReadinessProbe: true
```
Comment (roytman) [inline charts/fybrik/values.yaml:367]: why did you add `enableReadinessProbe` and `enableLivenessProbe` to OPA server only ?

**Review by roytman — COMMENTED** [review]
Great work, thanks. 
Do we have any test cases to check this PR ?

File: charts/fybrik/values.yaml (hunk 1 above)
Comment (revit13) [reply charts/fybrik/values.yaml:377]: OPA server implements the proves with http requests which require certificate verification when it uses tls as is used in notebook-test-readflow.tls. to avoid configuring kublet with the CA certificates this flag was added.
The rest of the deployments are not using probes with http requests so they can be executed anyway.

**Comment by revit13** [comment]
@roytman I did not include tests. I can try to add them.

File: charts/fybrik/values.yaml (hunk 1 above)
Comment (roytman) [reply charts/fybrik/values.yaml:377]: Can you add some explanation in the file ?

Comment (revit13) [reply charts/fybrik/values.yaml:377]: Sure, I mistakenly put it only in charts/fybrik/notebook-test-readflow.tls.values.yaml

**Review by roytman — APPROVED** [review]
